src/app/components/source-graphs/source-volume/source-volume.component.scss

```diff
@@ -0,0 +1,12 @@
+@import '../../../../styles/common';
+$side-label-size: 44px;
```

Comment:
There's already a global variable set on the `common` stylesheet, please use it instead as it's easier to manage any change.